spec/system/devises/invitation_spec.rb

```diff
@@ -2,10 +2,9 @@
 
 RSpec.describe 'User invitation', type: :system do
   describe 'Send invitation', type: :system do
-    let(:user) { create(:user) }
-
     it 'will send an invitation (pending)' do
-      sign_in(user)
+      admin = create(:user, roles: 'admin')
```

Comment:
I see you keep repeating creating a user with admin role, so you could create admin factory it, so you just call `create(:admin)`